styleguide/derek/templates/index.jade

```diff
@@ -70,6 +70,6 @@
   h5 Landing Page Templates
   ul
     li
-      a.red(href="derek/templates/landing-page/lockedContent-pdf") CTA Landing Page - Can be used for chat or form
```

Comment:
If we're removing this link, can the file be removed too?

Reply:
The file should stay for now since there won't be complete adoption of the new landing page when we push this. We should have an example available of the legacy version but I removed the link so we can encourage the use of the new one instead.

Reply:
👍